src/ext.ts

```diff
@@ -19,7 +23,7 @@ const columnExpressionItems = {
     translation: 'Object.Table.Columns.VisibilityCondition',
     defaultValue: { qv: '' },
     tid: 'visibilityCondition',
-    isExpression: (val) => typeof val === 'string' && val.trim().length > 0,
+    isExpression: (val: string) => typeof val === 'string' && val.trim().length > 0,
```

Comment:
I'm pretty sure val can be something else since the function checks if it is a string

Reply:
right, then any idea what other types could be?

Reply:
I guess yeah